.github/workflows/bicep_lint.yml

```diff
@@ -0,0 +1,30 @@
+name: Bicep Quality Assurance - Lint
+
+on:
+  pull_request:
+    branches:
+      - feat/stbr-infra
+    paths:
+      - '.github/workflows/bicep_az_deployment_group.yml'
+      - '.github/workflows/bicep_deploy.yml'
+      - '.github/workflows/bicep_workflow_feature.yml'
+      - 'infrastructure/resource_group_level/**'
```

Comment:
should this just be infrastructure? Right now I assume everything is resource_group_level but would assume if we ever have any bicep that's at a subscription level we would need it here

Reply:
Hm yes, good point